packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInteractiveInvalid.test.jsx

```diff
@@ -1,151 +1,145 @@
+  const handleIsInvalidChange = () => {
+    setIsInvalid(!isInvalid);
+  };
 
-  render() {
-    const field = (
-      <Field
-        id="default"
-        className={cx('form-field')}
-        label="Field Label"
-        help="Help Message"
-        error="Error Message"
-        isInvalid={this.state.isInvalid}
-      >
-        <div className={cx('field-content')}>Control Placeholder</div>
-      </Field>
-    );
+  const field = (
+    <Field
+      id="default"
+      className={cx('form-field')}
+      label="Field Label"
+      help="Help Message"
+      error="Error Message"
+      isInvalid={isInvalid}
+    >
+      <div className={cx('field-content')}>Control Placeholder</div>
+    </Field>
+  );
 
-    const optional = (
-      <Field
-        id="optional"
-        className={cx('form-field')}
-        label="Field Label"
-        help="Help Message"
-        error="Error Message"
-        isInvalid={this.state.isInvalid}
-        showOptional
-      >
-        <div className={cx('field-content')}>Control Placeholder</div>
-      </Field>
-    );
+  const optional = (
+    <Field
+      id="optional"
+      className={cx('form-field')}
+      label="Field Label"
+      help="Help Message"
+      error="Error Message"
+      isInvalid={isInvalid}
+      showOptional
+    >
+      <div className={cx('field-content')}>Control Placeholder</div>
+    </Field>
+  );
+
+  const required = (
+    <Field
+      id="required"
+      className={cx('form-field')}
+      label="Field Label"
+      help="Help Message"
+      error="Error Message"
+      isInvalid={isInvalid}
+      required
+    >
+      <div className={cx('field-content')}>Control Placeholder</div>
+    </Field>
+  );
+
+  const requiredHidden = (
+    <Field
+      id="required-hidden"
+      className={cx('form-field')}
+      label="Field Label"
+      help="Help Message"
+      error="Error Message"
+      isInvalid={isInvalid}
+      required
+      hideRequired
+    >
+      <div className={cx('field-content')}>Control Placeholder</div>
+    </Field>
+  );
 
-    const required = (
+  const hiddenLabel = (
+    <Field
+      id="label-hidden"
+      className={cx('form-field')}
+      label="Field Label"
+      help="Help Message"
+      error="Error Message"
+      isInvalid={isInvalid}
+      isLabelHidden
+    >
+      <div className={cx('field-content')}>Control Placeholder</div>
+    </Field>
+  );
+
+  const inline = (
+    <div>
       <Field
-        id="required"
+        id="inline-1"
         className={cx('form-field')}
-        label="Field Label"
-        help="Help Message"
+        label="Field Label - Spacing POC"
         error="Error Message"
-        isInvalid={this.state.isInvalid}
+        isInline
+        isInvalid={isInvalid}
         required
       >
         <div className={cx('field-content')}>Control Placeholder</div>
       </Field>
-    );
 
-    const requiredHidden = (
       <Field
-        id="required-hidden"
+        id="inline-2"
```

Comment:
why were these id's changed?

Reply:
The id was not changed, as seen here : https://github.com/cerner/terra-core/pull/3201/files#diff-ffc5fb5043d0efe4022dfec514e2701c55205d184173e75d716036d73a95aae1R58.
Looks like git did not map the difference between the changed files correctly.